ApollosProject/apollos-apps
# fix(pulse): update profile data for initials

## ✏️ Description
- updates profile data to pull initials correctly from first and last name



## 🔬 To Test
1. Log in to app with user without profile image
2. View profile details
3. View pulse card user details
4. Initials should match

## 📸 Screenshots

Profile
![image](https://user-images.githubusercontent.com/7557597/206224818-8b5a58a1-911b-4bba-9830-3174d90ffa0d.png)
Pulse card
![image](https://user-images.githubusercontent.com/7557597/206224900-d967a7fa-e483-4975-9a4f-3ee0596fb480.png)


## ⚠️ To-do Before Merge

N/A

## 📝 Checklist
